client/wfnews-war/src/main/angular/src/app/components/report-of-fire/compass-page/rof-compass-page.component.ts

```diff
@@ -42,12 +42,15 @@ ngOnInit(): void {
 
 async getOrientation() {
   try{
+    let self = this;
     const requestPermission = (DeviceOrientationEvent as unknown as DeviceOrientationEventiOS).requestPermission;
     const iOS = typeof requestPermission === 'function';
     if (iOS) {
     const response = await requestPermission();
         if (response === "granted") {
```

Comment:
When testing strings, it's always a good idea to ignore case, unless case is important (in this situation, I assume Granted and granted would mean the same thing)